templates/html_report/css/style.css

```diff
@@ -578,7 +578,7 @@ table tr td {
     position:relative;
 }
 .js-sort-table td {
-    padding:.125em .325em .125em 0;
+    vertical-align:baseline;
```

Comment:
This one removes an element jumping which happens when you click on a class name.